ftva_lab_data/fixtures/sample_data.json

```diff
@@ -316,5 +316,47 @@
       "general_entry_cataloged_by": "",
       "notes": ""
     }
+  },
```

Comment:
Since item statuses are real data that should always be loaded, please move these into a separate `item_statuses.json` fixture.  Then for the test case, load both fixtures.